react/fields/Dropdown/Dropdown.less

```diff
@@ -0,0 +1,111 @@
+@import (reference) "~seek-style-guide/theme";
+
+.root {
+  display: inline-block;
+  position: relative;
+}
+
+.label {
+  .standardText();
+  display: inline-block;
+  margin-bottom: @grid-row-height;
+}
+
+.dropdown {
+  .touchableText();
+  width: 100%;
+  display: block;
+  height: @grid-row-height * 5;
+  padding: 0 @field-gutter-width;
+  background: @sk-white;
+  border: @field-border-width solid @sk-mid-gray-light;
+  border-radius: @field-border-radius;
+  color: @sk-charcoal;
+  background-color: @sk-white;
+  cursor: pointer;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+
+  &:not([readonly]):not([disabled]):focus,
+  .rootFocus & {
+    outline: none;
+    border-color: transparent;
+    .focus();
+  }
+
+  .invalid & {
+    border-color: transparent;
+    box-shadow: 0 0 0 @field-border-width @sk-pink;
+
+    &:not([readonly]):not([disabled]):focus,
+    .rootFocus & {
+      border-color: transparent;
+      box-shadow: 0 0 0 @field-border-width @sk-pink;
+    }
+  }
+
+  &.blankValueSelected {
+    color: @sk-mid-gray;
+  }
+
+  &:first-child {
+    margin-right: @grid-gutter-width / 2;
+  }
+
+  &:last-child {
+    margin-bottom: (@grid-row-height * 3);
+  }
```

Comment:
these `first-child` and `last-child` rules look like implementation detail rather than `Dropdown` styles, is that right?